.github/workflows/testing.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
     - uses: actions/setup-python@v2
       with:
-        python-version: "3.x"
+        python-version: "3.7"
```

Comment:
Does the 3.x imply that many versions of Python would be tested? Is it possible to test the last three trailing versions minus the current (which is typically always problematic)?

Reply:
It's a sort of "wildcard" that matches the latest minor python3 version in github CI.
Since @pmav99 suggested to use the lowest available, I changed it to `3.7`